closingd/closing.c

```diff
@@ -80,6 +83,92 @@ static u64 one_towards(u64 target, u64 value)
 	return value;
 }
 
+static void handle_ping(const u8 *msg,
```

Comment:
Cannot we factor out this function to `common/ping.c`?  I dislike multiple functions with different contents having the same name.  In addition, the `handle_ping` in openingd and closingd look very very similar (possibly except for `PEER_FD` but that can be passed in).  Only the `handle_ping` in channeld is different because it enqueues the `pong` reply rather than sending it synchronously (but maybe sending it synchronously is acceptable to reduce code duplication...?)

Reply:
Yeah, as commit msg says, they're just similar enough to be annoying.  I think trying to merge this is a future TODO.